server/accounts.go

```diff
@@ -3008,6 +3073,17 @@ func (dr *DirAccResolver) Start(s *Server) error {
 			}
 		}); err != nil {
 		return fmt.Errorf("error setting up pack request handling: %v", err)
+	} else if _, err = s.sysSubscribe(accListReqSubj,
+		// respond to list requests with one message containing all account ids
+		func(_ *subscription, _ *client, _, reply string, _ []byte) {
+			handleListRequest(dr.DirJWTStore, s, reply)
+		}); err != nil {
```

Comment:
that threw me off, had to have a second look to see that this was the `err` for a failed s.sysSubscribe()

Reply:
moved the comment ahead of if and the function on the sysSubscribe line